tests/test_driver.py

```diff
@@ -110,20 +110,32 @@ def test_send(self, driver, alice_privkey, unsigned_transaction):
         assert sent_tx == fulfilled_tx
 
 
-class TestUnspentsEndpoint:
+class TestOutputsEndpoint:
 
-    @mark.skip
-    def test_get_unspents(self, carol_pubkey, driver,
-                          persisted_alice_transaction,
-                          persisted_carol_bicycle_transaction,
-                          persisted_carol_car_transaction):
+    def test_get_outputs(self, carol_pubkey, alice_pubkey, driver,
+                         persisted_alice_transaction,
+                         persisted_carol_bicycle_transaction,
+                         persisted_carol_car_transaction):
         # FIXME The sleep, or some other approach is required to wait for the
         # transaction to be available as some processing is being done by the
         # server.
         sleep(1.5)
-        unspents = driver.unspents.get(carol_pubkey)
-        assert len(unspents) == 2
-        assert '../transactions/{id}/conditions/0'.format_map(
-            persisted_carol_bicycle_transaction) in unspents
-        assert '../transactions/{id}/conditions/0'.format_map(
-            persisted_carol_car_transaction) in unspents
+        outputs = driver.outputs.get(carol_pubkey)
+        assert len(outputs) == 2
+        assert '../transactions/{id}/outputs/0'.format_map(
+            persisted_carol_bicycle_transaction) in outputs
+        assert '../transactions/{id}/outputs/0'.format_map(
+            persisted_carol_car_transaction) in outputs
+
+    def test_get_unspents_outputs(self, carol_pubkey, dimi_pubkey, driver,
+                                  persisted_carol_bicycle_transaction,
+                                  persisted_carol_car_transaction,
+                                  persisted_transfer_carol_car_to_dimi):
+        # FIXME The sleep, or some other approach is required to wait for the
+        # transaction to be available as some processing is being done by the
+        # server.
+        sleep(1.5)
+        outputs = driver.outputs.get(carol_pubkey, unspent=True)
+        assert len(outputs) == 1
+        assert '../transactions/{id}/outputs/0'.format_map(
+            persisted_carol_bicycle_transaction) in outputs
```

Comment:
Mmm, I would start the test by asserting that Carol has 2 transactions in total, and then test that with the `unspent` flag, it's only one left.